catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerTest.java

```diff
@@ -183,11 +183,11 @@ protected AdminSourceHelper getHelper() {
                 // Mock out the metatypes
```

Comment:
Is it possible to add a test that would verify the SchematronValidationService is filtered out based on the `exludeAsSource`?